ckan/views/util.py

```diff
@@ -39,7 +39,19 @@ def custom_form_fields() -> str:
     )
 
 
+def csrf_input() -> str:
+    '''
+    Render a hidden CSRF input field that refreshes automatically
+    so that users won't get an expired token error.
+    '''
+    # refresh just before expiration (enforce min in case of misconfiguration)
+    return base.render('snippets/csrf_input.html', {
```

Comment:
When calling this function. 

Can we either, add either;
1. a nonce param so its always unique
2. set to sensitive ``no-cache`` response cache level
3. change to a POST and ensure csrf protections are not on said function.